static/app/views/replays/replayTable/tableCell.tsx

```diff
@@ -415,7 +411,6 @@ const StyledIconDelete = styled(IconDelete)`
   margin: ${space(0.25)};
 `;
 
-// Need to be full width for StringWalker to take up full width and truncate properly
 const UserBadgeFullWidth = styled(UserBadge)`
   width: 100%;
 `;
```

Comment:
Can we remove UserBadgeFullWidth and use `<UserBadge>` directly now?

Reply:
i think so!